src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100..900;1,100..900&family=Space+Grotesk:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="iMac picture"
+      />
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__raiting">
+        <div class="card__stars">
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+          <span class="card__star"></span>
+        </div>
+        <div class="card__reviews">
+          <p>Reviews: 5</p>
+        </div>
+      </div>
+      <div class="card__value">
+        <p class="card__price">Price:</p>
+        <p class="card__number">$2,199</p>
+      </div>
+      <a href="#">
+        <button
+          class="card__button"
+          type="button"
+          data-qa="hover"
+        >
+          Buy
+        </button>
+      </a>
```

Comment:
```suggestion

        <button
          class="card__button"
          type="button"
          data-qa="hover"
        >
          Buy
        </button>

```
You need to make a choice and use `button` OR `a` here depending on the functionality you want to achieve